include/ofi_mr.h

```diff
@@ -146,9 +153,11 @@ struct ofi_mem_monitor {
 	void (*stop)(struct ofi_mem_monitor *monitor);
 	int (*subscribe)(struct ofi_mem_monitor *notifier,
 			 const void *addr, size_t len,
+			 void **ipc_handle,
 			 union ofi_mr_hmem_info *hmem_info);
 	void (*unsubscribe)(struct ofi_mem_monitor *notifier,
 			    const void *addr, size_t len,
+			    void **ipc_handle,
```

Comment:
Do we need to modify subscribe/unsubscribe?  Won't these be no-ops for the IPC case, and we always need to call valid() instead?

Can we separate the changes needed to support IPC handles on the exporter side from the importer side?

Reply:
`subscribe` is not a no-op for IPC, this is the function

```
static int ipc_handle_monitor_subscribe(struct ofi_mem_monitor *monitor, const void *addr,
					size_t len, void **ipc_handle,
					union ofi_mr_hmem_info *hmem_info)
{
        memcpy((void **)&hmem_info->ipc_handle, ipc_handle, IPC_HANDLE_SIZE);
	return FI_SUCCESS;
}


```

Reply:
>Can we separate the changes needed to support IPC handles on the exporter side from the importer side?

Just to make sure I understand, currently the `ofi_import_monitor` follows the same API with `ofi_mem_monitor`, so I need to modify the API for `ofi_import_monitor_valid` and `ofi_import_monitor_subscribe` . This PR does not change the `export_ops` for the exporter side, just ignores the `ipc_handle` when calling the `export_ops` inside `ofi_import_monitor_*`.

Are you suggesting making the changes on import_monitor API as a separate commit, or something else?